src/styles/utils/variables.scss

```
@@ -0,0 +1,10 @@
+$colorPrimary: #00acdc;
+$colorAccent: #060b35;
+$colorText: #616070;
+
+$gapX: 50px;
+$gapY:48px;
```

mock has these exact measures, yet your tests might fail by +2px on each card (borders)
if tests fails try making
```suggestion
$gapX: 48px;
$gapY:46px;
```